pyproject.toml

```
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["flit_core >=2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.metadata]
+author = "Ciencia de Datos • GECI"
+author-email = "ciencia.datos@islas.org.mx"
+classifiers = [ "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",]
+description-file = "README.md"
+home-page = "https://github.com/IslasGECI/clean-enoa.py"
```

¿Esto entrará al equipo de Ciencia de Datos de GECI?

```suggestion
home-page = "https://github.com/nepito/clean-enoa.py"
```